OWML.ModHelper.Menus/ModPopupMenu.cs

```diff
@@ -58,7 +55,7 @@ public virtual void Open()
         {
             if (Menu == null)
             {
-                _console.WriteLine("Warning: can't open menu, it doesn't exist.");
+                Console.WriteLine("Warning: can't open menu, it doesn't exist.");
```

Comment:
Isn't this now just using System.Console...?

Reply:
now MyConsole just in case